### PR TITLE
Update CHANGLEOG and Readme.md

### PR DESCRIPTION
**Pull request by reazulhoque**
As title.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,31 @@ example, you can try the `vector addition` example as follows:
...
+with numba_dppy.offload_to_sycl_device(dpctl.SyclQueue()):
+    X = np.arange(10)
+    Y = np.arange(10)
+    div_kernel[10, numba_dppy.DEFAULT_LOCAL_SIZE](Y, X, 5)
+    D = X//5
+    print(Y, D)
+```
+
+To bypass this issue we need latest `llvm-spirv` tool. Users can get it by explicitly installing `dpcpp` Conda package. The `llvm-spirv` tool is packaged as part of the `dpcpp` Conda package.
+
```
Comment (diptorupd) [inline README.md:68]: will the conda `llvm-spirv` package need to be uninstalled first?

File: README.md [hunk 2]
```diff
@@ -44,6 +44,31 @@ example, you can try the `vector addition` example as follows:
...
+`conda install dpcpp_linux-64`
```
Comment (diptorupd) [inline README.md:69]: and windows?

File: README.md (hunk 1 above)
Comment (reazulhoque) [reply README.md:68]: No.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.17.0] - 2021-10-xx
+## [0.17.0] - 2021-11-03
```
Comment (diptorupd) [inline CHANGELOG.md:8]: We need to add the RC tag changes here as well. Once you do so, I will go over them in one pass and suggest edits to make the changelog entries clearer.

File: README.md (hunk 2 above)
Comment (reazulhoque) [reply README.md]: Updated.

**Review by diptorupd — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (PokhodenkoSA) [reply CHANGELOG.md:8]: No.
RC is pre-release tags.

Comment (PokhodenkoSA) [reply CHANGELOG.md:8]: As we do not recommend use dev, alpha, beta, rc marked packages in production - it is not a release.

File: README.md [hunk 4]
```diff
@@ -44,6 +44,32 @@ example, you can try the `vector addition` example as follows:
 python numba_dppy/examples/sum.py
 ```
 
+# Known Issue
```
Comment (PokhodenkoSA) [inline README.md:47]: I think there should be a sections about limitations in documentation.
I.e. there is example for device function where limiataions are - todo - https://github.com/IntelPython/numba-dppy/blame/main/docs/user_guides/kernel_programming_guide/device-functions.rst#L29
I think this limitation is related to Python features support so the better document is https://github.com/IntelPython/numba-dppy/blob/main/docs/user_guides/kernel_programming_guide/supported-python-features.rst

Comment (PokhodenkoSA) [reply README.md:47]: @reazulhoque 
You can keep the section `Known Issues` and place the link to corresponding docs inside it. 
It is more scalable approach as we can not place all limitations in README.

**Pull request merged**